fiftyone/core/odm/dataset.py

```diff
@@ -480,21 +493,81 @@ class DatasetDocument(Document):
     brain_methods = DictField(ReferenceField(RunDocument))
     evaluations = DictField(ReferenceField(RunDocument))
 
+    def get_saved_views(self):
+        saved_views = []
+        for view_doc in self.saved_views:
+            if not isinstance(view_doc, DBRef):
+                saved_views.append(view_doc)
+            else:
+                logger.warning(
```

Comment:
```suggestion
                oid = getattr(ref, "id", "")
                col = getattr(ref, "collection", "")
                created = getattr(ref.id, "generation_time", "") if oid else ""
                logger.error(
                    f"Could not resolve ObjectId('{oid}') from '{col}' collection created at {created}"
                )
`````

Can we include something more informative like the above in the logging message? I think it will help with diagnosing the problem

Reply:
The user could just pass `dry_run=True` to the relevant patch method to see the offending IDs. Let's just stick with that for now